src/common/utils/gpo.h

```diff
@@ -305,4 +306,8 @@ namespace powertoys_gpo {
         return getConfiguredValue(POLICY_ALLOW_EXPERIMENTATION);
     }
 
+    inline gpo_rule_configured_t getConfiguredEnvironmentVariablesEnabledValue()
+    {
+        return getConfiguredValue(POLICY_CONFIGURE_ENABLED_ENVIRONMENT_VARIABLES);
```

Comment:
If https://github.com/microsoft/PowerToys/pull/28731 gets merged the way of implementing the utility policies changes a bit. Please be aware of this change.

Reply:
PR is merged. Please update your new code.

Reply:
thanks. I'll update it

Reply:
done